dev/src/telemetry/trace-util.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright 2024 Google LLC. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Span} from './span';
+
+export interface Attributes {
+  [attributeKey: string]: AttributeValue | undefined;
+}
+export declare type AttributeValue =
+  | string
+  | number
+  | boolean
+  | Array<string>
+  | Array<number>
+  | Array<boolean>;
+
+/**
+ * Span names for instrumented operations.
+ */
+export const SERVICE = 'google.firestore.v1.Firestore/';
```

Comment:
No `SPAN_NAME_RUN_QUERY`, `SPAN_NAME_RUN_AGGREGATION_QUERY`,`SPAN_NAME_TRANSACTION_BEGIN` in node?

Reply:
The Node SDK has been updated such that it never needs to send a BeginTransaction rpc to the server. (see https://github.com/googleapis/nodejs-firestore/pull/2017).

Regarding using the span name as method name, I'm hesitant to do it. For example we'd have something like:

```js
       const methodName = SPAN_NAME_RUN_AGGREGATION_QUERY;
        const backendStream = await firestore.requestStream(
          methodName,
          /* bidirectional= */ false,
          request,
          tag
        );
```
because the person reading the code may be confused why we're passing a tracing span name to the requestStream function. Or you may wonder if it's really needed if tracing is OFF. This is in fact the rpc method [name](https://github.com/googleapis/nodejs-firestore/blob/f73e28b69be3ff9f870055af80683f12048549e8/dev/src/types.ts#L98). There are minor differences between Java and Node in this aspect. For example, in node both query and aggregate query call the same underlying "requestStream" function, but in Java there are separate classes for handling query vs aggregate query.

Reply:
I agree. the above code should not use span constant as it is not tracing related. But, we should use it to creating span. like the code change here:
<img width="1304" alt="Screenshot 2024-07-24 at 10 06 20 AM" src="https://github.com/user-attachments/assets/928af704-d49e-4ab4-b7eb-4df8f3522081">

I prefer not to extract the string to `methodName`, and use the `
SPAN_NAME_RUN_AGGREGATION_QUERY` inside the `addEvent`. 
Even tho the string used are the same, but they are serving the different purposes.

Reply:
Oh I see what you mean.

Reply:
Done